apps/console/src/public/resources/iframe.html

```diff
@@ -0,0 +1,46 @@
+<!--
+~    Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+~
+~    This software is the property of WSO2 LLC. and its suppliers, if any.
+~    Dissemination of any information or reproduction of any material contained
+~    herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+~    You may not alter or remove any copyright or other notice from copies of this content."
+-->
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Impersonate Iframe</title>
```

Comment:
No need to have a title, because this is/will never be visible to the end user.

Reply:
Removed.

Reply:
fixed with e55b3e89f34988d5d2d50cb8e10065252294bf21